src/sync.js

```diff
@@ -202,6 +204,8 @@ function syncKinto(client, credentials) {
       });
     })
     .then(syncResult => {
+      lastSyncTimestamp = new Date().getTime(); // eslint-disable-line no-undef
```

Comment:
This is a bit tedious, we shoud use the server timestamp here. I think Kinto.js is supposed to keep it :/
https://github.com/Kinto/kinto.js/blob/master/src/collection.js#L286

Reply:
Not yet but that's something we should definitely investigate.

Reply:
That's probably why kinto.js is not handling this by itself and need our dirty patch 😓.

Reply:
@vladikoff fyi, **PR is working** but this is the dirty solution I had to implement. We keep track in sync.js instead of Kinto.Collection object which return for unknown reason lastModified as `NaN`. Except that point, it's 100% handled by kinto as expected *(no extra fetch as discussed)*.

We might have an issue having some gap between real sync and `new Date()` (could we save that date before requesting so worst case we have extra redundant data from those millisecond on next sync ?).

Once again we're dirty and it shows something is wrong in our process with Kinto. I would suggest to open an extra bug and keep working on it when having more time.